src/platform/nrfconnect/shell/ChipShell.cpp

```diff
@@ -0,0 +1,197 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/server/OnboardingCodesUtil.h>
+#include <platform/CHIPDeviceLayer.h>
+#include <support/CodeUtils.h>
+
+#include "shell/shell.h"
```

Comment:
I don't see this header, is it referring to the renamed 'shell_core.h'?

Reply:
It's a header from Zephyr. That's `shell.h` from CHIP was renamed to `shell_core.h` to solve the file name collission :)